EasyPost.Tests/UnitTest.cs

```diff
@@ -0,0 +1,85 @@
+using System;
+using System.Threading.Tasks;
+
+namespace EasyPost.Tests
+{
+    /// <summary>
+    ///     Base class for all unit tests.
+    ///     Sets up all available client versions for VCR and non-VCR requests.
+    /// </summary>
+    public class UnitTest : IDisposable
+    {
+        private readonly TestUtils.VCR? _vcr;
+
+        protected Client Client;
+
+        private string? _cleanupId;
+
+        /// <summary>
+        ///     An asynchronous function executed after every test.
+        ///     Function should have a string (ID) passed in, and should return a boolean.
+        /// </summary>
+        protected Func<string, Task<bool>>? CleanupFunction { get; set; }
+
+#pragma warning disable CS8618
+        protected UnitTest(string groupName, TestUtils.ApiKey apiKey = TestUtils.ApiKey.Test) => _vcr = new TestUtils.VCR(groupName, apiKey);
+#pragma warning restore CS8618
```

Comment:
Question/Issue: why not fix the actual warning issue instead of disabling it?

Something like https://stackoverflow.com/a/60702710/6064135?

Reply:
Because the error is presumptious. It thinks that the `Client` object is not being declared, and suggesting that it be nullable. If that were the case, it would require slapping nullability checks into every single unit test for something that I know is not nullable (`Client` is set shortly after initialization of the unit test and is guaranteed to be set by the time the unit test executes). It's a case of me being more aware of what the code is doing than the IDE is.

Reply:
Perfect explanation, can you add a simple comment detailing the same in code? That way we know why we did the thing later on when we all inevitably forget the reasoning.

Reply:
Resolved.